exercises/nth-prime/runtests.jl

```diff
@@ -0,0 +1,27 @@
+using Test

```

Comment:
```suggestion
# canonical data version: 2.1.0

using Test
```

This makes it easier to maintain and update in the future (see `CONTRIBUTING.md`) and otherwise the [dashboard](https://tracks.exercism.io/julia) can't detect this version.